### ISSUE TITLE
[Plugin Request] - Integrated Terminal in lite

### ISSUE THREAD

**Issue by xvishaldongre** [rxi/lite-plugins#137]
Hi there, It would be really nice if someone can create a **Integrated Terminal** like VS Code.

**Comment by BenStigsen** [comment]
Agreed. There's already the [console](https://github.com/rxi/console) plugin made by rxi. But it would be nice to customize build commands depending on the language extension or to optionally run a build file (build.bat, build.sh, ....) if chosen.

**Comment by drmargarido** [comment]
Also filtering by extension may be done inside the command that is being run.
